[DOC] Complete the profiling section (#83)
perses/website: docs/blog/posts/v0.52.0.md

# Release v0.52.0

## New profiling plugins

This release provides the support of Continuous Profiling through two plugins:

> - **Pyroscope**: a datasource plugin that allows you to retrieve and transform data from a Pyroscope server.

Reviewer: ```suggestion
- **Pyroscope**: a datasource plugin for retrieving and handle data from a Pyroscope server.
```

> - **Flame Chart**: a visualization plugin which contains 3 panles: Flame Graph, Table View and Series Chart.

Reviewer: ```suggestion
- **Flame Chart**: a visualization plugin which contains 3 panels: Flame Graph, Table View and Series Chart.
```

### What is Continuous Profiling?

Continuous Profiling is a new observability signal, complementing the traditional logs, metrics, and traces. This innovative approach provides a way of continuously measuring applications' resource consumption with code-level granularity and low overhead. This signal generates excitement in the community for its ability to provide in-depth visibility into production performance of applications.

### Panels

#### Flame Graph

> The flame graph panel offers a simple and intuitive way to represent performance data, providing a hierarchical view that lets you zoom in on any profile for detailed analysis. It helps undertand resource allocation (CPU, Memory, etc.) and bottlenecks within an application.

Reviewer: ```suggestion
The flame graph panel offers a simple and intuitive way to represent performance data, providing a hierarchical view that let you zoom in on any stack for a detailed analysis. It helps to understand resource allocations (CPU, Memory, ...) and bottlenecks within an application.
```

![Flame Graph Panel](../../assets/images/blog/v052/flame-graph-panel.png)

#### Table View

> This panel displays the data in a table format. It allows you to sort the data, for example in descending order, to see which functions consume the most resources. A search bar also lets you filter the table’s content to focus on a fonctions whose names contain a specific string.

Reviewer: ```suggestion
This panel displays the profile in a table format. It allows you to sort stacks, for example in descending order, to see which function consume the most resources. A search bar is also present for filtering the table and highlight stacks in the flame graph.
```

![Table View Panel](../../assets/images/blog/v052/table-view-panel.png)

#### Series Chart

When retrieving profiles from the Pyroscope server, they include timeline data showing how the total amount of resources consumed evolves over time, depending on the selected profile type. This panel allows you to visualize this data in order to identify spikes in resource consumption (CPU, memory, etc.) and focus on them by zooming in on the corresponding time range.

![Series Chart Panel](../../assets/images/blog/v052/series-chart-panel.png)

> In next releases, we plan to add a Diff Flame Graph feature to compare two Flame Graphs, and to use the OpenTelemetry profiling datamodel for better compatibility of this plugin with other plugins using the same datamodel. Stay tuned.

Reviewer: Not planned in the short term, it should be removed
